src/pydvl/value/result.py

```diff
@@ -499,9 +499,9 @@ def __add__(self, other: "ValuationResult") -> "ValuationResult":
 
         """
         # empty results
-        if len(self.values) == 0:
+        if len(self.values) == 0 or np.all(self.counts == 0):
```

Comment:
This seems reasonable but is likely to break things. Check the tests for this class and add one for this or the previous behaviour. If it doesn't break anything, then you need to document the behaviour in the docstring.